db/postgres.go

```diff
@@ -199,39 +199,39 @@ func (p *pgClient) ExistingMetricsTimespan() (time.Time, time.Time, error) {
 }
 
 func (p *pgClient) WriteTodaysMetrics(counts map[string]api.RelayCounts, countsOrigin map[string]api.RelayCounts, latencies map[string][]api.Latency) error {
-	err := p.WriteTodaysUsage(counts, countsOrigin)
```

Comment:
if you're already making a context would be better if you take advantage of it, like a deadline, otherwise although necessary by the SQL API, you're not doing much with it

Reply:
True, I'll add a comment for the future update (we're hardcoding the time zone 😬)